.github/workflows/ci.yaml

```diff
@@ -265,7 +269,7 @@ jobs:
 
     - name: Alpine setup
       if: startsWith(matrix.container, 'alpine')
-      run: apk add git bash shadow sudo curl
+      run: apk add git bash shadow sudo curl tar
```

Comment:
Cache save was failing on Alpine due to busybox tar

https://github.com/chevah/compat/actions/runs/11202300894/job/31138241218?pr=700#step:24:4

Reply:
ACK